src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,33 @@
+import duke.Deadline;
+import duke.Task;
+import duke.TaskList;
+import duke.Todo;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class TaskListTest {
+    @Test
+    public void testEmptyTaskList() {
```

Comment:
Might want to reconsider the naming of testing methods (refer to [coding standard](https://se-education.org/guides/conventions/java/intermediate.html)).